apps/web/components/FeedbackFormCard.tsx

```diff
@@ -38,6 +38,13 @@ export const FeedbackFormCard: React.FC<Props> = ({ id, address }) => {
   const openModal = () => setIsModalOpen(true);
   const closeModal = () => setIsModalOpen(false);
 
+  const truncateDescription = (description: string | undefined) => {
+    if (description && description.length > 30) {
+      return description.slice(0, 30).trim() + "...";
```

Comment:
Note for you next time. tailwind `text-ellipsis`:

https://tailwindcss.com/docs/text-overflow